### PR TITLE
Fix api-docs anchor link vertical alignment

### LINKED ISSUE #4635: Docs anchor links are offset incorrectly
When you click on an anchor link for any FAQ or API endpoint, the page scrolls a few pixels past the title of the section. It's worse on mobile but bad everywhere.

doc-headings

### PR DESCRIPTION
**Pull request by hunicus**
Fixes #4635 more thoroughly, including mobile.

### CODE REVIEW

File: frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -270,7 +270,7 @@ if (browserWindowEnv && browserWindowEnv.BASE_MODULE === 'liquid') {
   imports: [RouterModule.forRoot(routes, {
     initialNavigation: 'enabledBlocking',
     scrollPositionRestoration: 'enabled',
-    anchorScrolling: 'enabled',
+    anchorScrolling: 'disabled',
```
Comment (nymkappa) [inline frontend/src/app/app-routing.module.ts:273]: Don't we use this for other components tho @softsimon @mononaut

Comment (hunicus) [reply frontend/src/app/app-routing.module.ts:273]: There's the vin/vout anchor links (e.g. https://mempool.space/tx/b00871cec33d359da8d23a0dc6fd685ad776edb5a5d89e8de776ef4e296dc51d#vin=35) but those don't scroll.

There's also the anchor links on the about page but those seem broken to me on mempool.space already and could be fixed to use the mechanism in this PR.

**Comment by hunicus** [comment]
Added v3.0 milestone because otherwise API docs will be broken on all Raspberry Pi nodes, since they run the open-source-only build.

**Review by nymkappa — APPROVED** [review]
Tested ACK @ e408fbd8d6b9b138810ea1e39c9f7dfbc73b6fe3

**Pull request merged**